ansible-koha/files/zebraidx.service

```diff
@@ -1,12 +1,14 @@
 [Unit]
 Description=Zebra daemon for Koha
+Requires=mariadb.service
+After=mariadb.service
 
 [Service]
 Type=simple
 User=koha
 PermissionsStartOnly=true
-ExecStartPre=/usr/bin/chmod -R 777 /var/lock/koha/zebradb/ /var/run/koha/zebradb/ /var/lib/koha/zebradb/
```

Comment:
This was needed! I'm moving it to its own task. I was wondering where all those zebra failures were coming from. :)